MAUI/Calendar/localization.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Localization in MAUI Calendar | Syncfusion
+description: Learn here all about Localization feature of Syncfusion MAUI Calendar(SfCalendar) widget and more. 
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Localization in MAUI Calendar (SfCalendar)
+
+By default, the calendar widget supports US English localizations. You can change to other languages by adding the `Globalization` namespace to your application.
+
+{Application culture can be changed by setting `CurrentUICulture.` in `App.xaml.cs` file.
```

Comment:
The application culture can be changed by setting `CurrentUICulture.` in the `App.xaml.cs` file.

Check the brace